[UPDATE] Update and expand unit 1 (#882)
huggingface/course: chapters/en/chapter1/2.mdx

NLP isn't limited to written text though. It also tackles complex challenges in speech recognition and computer vision, such as generating a transcript of an audio sample or a description of an image.

## What are Large Language Models (LLMs)?[[what-are-llms]]

> Large Language Models (LLMs) are a type of artificial intelligence model designed to understand, generate, and manipulate human language. They represent a significant advancement in the field of natural language processing (NLP).
> 
> <Tip>
> 
> A large language model is an AI system trained on massive amounts of text data that can understand and generate human-like text, recognize patterns in language, and perform a wide variety of language tasks without task-specific training.
> 
> </Tip>

Reviewer: I think even this can be simplified to avoid some redundancy

```suggestion
A large language model (LLM) is an AI model trained on massive amounts of text data that can understand and generate human-like text, recognize patterns in language, and perform a wide variety of language tasks without task-specific training. They represent a significant advancement in the field of natural language processing (NLP).
```
